frontend/components/site/SiteSettings/BasicSiteSettings.jsx

```diff
@@ -45,15 +46,13 @@ export const BasicSiteSettings = ({
             live URL
           </a> to ensure the site builds correctly.
         </p>
-        <label htmlFor="defaultBranchInput">
-          Branch name:
-        </label>
-        <Field
-          component="input"
+        <BranchField
+          label="Branch name:"
           type="text"
           id="defaultBranchInput"
           name="defaultBranch"
           className="form-control"
```

Comment:
What do you think about either setting a blank placeholder on the demoBranch field or adding a non-blank one here?

Reply:
My feeling here is that its never going to be blank in the above example, I can't really think of any non-broken page state where the site isnt going to have a primary branch. We can duplicate the demoBranch value if it seems safer.

Reply:
Yeah, that's a fair point. I don't feel strongly about it.

Reply:
I ended up changing, it is very simple, like you said, and it doesn't hurt anything 😄